doc/cephfs/client-auth.rst

```diff
@@ -153,3 +148,93 @@ The optional ``{network/prefix}`` is a standard network name and
 prefix length in CIDR notation (e.g., ``10.3.0.0/16``).  If present,
 the use of this capability is restricted to clients connecting from
 this network.
+
+File system Information Restriction
+===================================
+
+If desired, the monitor cluster can present a limited view of the file systems
+available. In this case, the monitor cluster will only inform clients about
+file systems specified by the administrator. Other file systems will not be
+reported and commands affecting them will fail as if the file systems does
+not exists.
+
+Consider following example. The Ceph cluster has 2 FSs::
+    $ ceph fs ls
+    name: cephfs, metadata pool: cephfs_metadata, data pools: [cephfs_data ]
+    name: cephfs2, metadata pool: cephfs2_metadata, data pools: [cephfs2_data ]
+
+But we authorize client ``someuser`` for only one FS::
+    $ ceph auth add client.someuser mon 'allow rw fsname=cephfs' mds 'allow rw' osd 'allow rw'
```

Comment:
Why not use the `fs authorize` interface in the doc?

Reply:
Because we want to demonstrate the impact on `fs ls` and therefore the client needs MON caps, not MDS caps.

Reply:
Modified `fs authorize`'s code to assign MON caps too, the doc now never uses `auth add` now.